### PR TITLE
[#1406] When running tests, report test ns in minibuffer messages.

### LINKED ISSUE #1406: The test ns should be displayed in the minibuffer test status message
When customizing `cider-test-infer-test-ns` with a custom function, it would be helpful to know
which tests ran when there might be multiple places. This would also be consistent with
the test report which prominently features the test ns.

### PR DESCRIPTION
**Pull request by Fabricationatorishnicatorist**
Closes #1406

### CODE REVIEW

File: cider-test.el [hunk 1]
```diff
@@ -434,7 +434,7 @@ This uses the Leiningen convention of appending '-test' to the namespace name."
 Upon test completion, results are echoed and a test report is optionally
 displayed. When test failures/errors occur, their sources are highlighted."
   (cider-test-clear-highlights)
-  (message "Testing...")
+  (message "Testing %s..." ns)
```
Comment (bbatsov) [inline cider-test.el]: Technically speaking you're `running` the tests in `ns`, you're `testing` another ns (the one with the actual implementations). Maybe both namespaces should be mentioned in the message (or it should be reworded).

File: cider-test.el [hunk 2]
```diff
@@ -325,12 +325,12 @@ With the actual value, the outermost '(not ...)' s-expression is removed."
 
 ;;; Summary echo
 
-(defun cider-test-echo-summary (summary)
+(defun cider-test-echo-summary (summary ns)
   "Echo SUMMARY statistics for a test run."
```
Comment (bbatsov) [inline cider-test.el]: NS should appear in the docstring.

File: cider-test.el (hunk 1 above)
Comment (Fabricationatorishnicatorist) [inline cider-test.el]: Reworded.

File: cider-test.el (hunk 2 above)
Comment (Fabricationatorishnicatorist) [inline cider-test.el]: Done.

File: cider-test.el [hunk 3]
```diff
@@ -325,12 +325,12 @@ With the actual value, the outermost '(not ...)' s-expression is removed."
...
 
-(defun cider-test-echo-summary (summary)
-  "Echo SUMMARY statistics for a test run."
+(defun cider-test-echo-summary (summary ns)
+  "Echo SUMMARY statistics for a test run of tests in NS."
   (nrepl-dbind-response summary (test fail error)
     (message
      (propertize
-      (format "Ran %s tests. %s failures, %s errors." test fail error)
+      (format "Ran %d tests from %s. %s failures, %s errors." test ns fail error)
```
Comment (bbatsov) [inline cider-test.el]: Just noticed that there are more %s here that should be %d - failures and errors.

File: cider-test.el [hunk 4]
```diff
@@ -434,7 +434,7 @@ This uses the Leiningen convention of appending '-test' to the namespace name."
 Upon test completion, results are echoed and a test report is optionally
 displayed. When test failures/errors occur, their sources are highlighted."
   (cider-test-clear-highlights)
-  (message "Testing...")
+  (message "Running tests from %s..." ns)
```
Comment (bbatsov) [inline cider-test.el]: I think it'd be better to use `in` instead of `from`.

Comment (bbatsov) [inline cider-test.el]: I'd also use some different face for the namespace.

File: cider-test.el (hunk 3 above)
Comment (Fabricationatorishnicatorist) [inline cider-test.el]: Done.

File: cider-test.el (hunk 4 above)
Comment (Fabricationatorishnicatorist) [inline cider-test.el]: Changed to "in". You'll have to deal with the faces yourself if you consider that critical for merging, sorry.

Comment (Fabricationatorishnicatorist) [inline cider-test.el]: But I added some color, anyway.

**Comment by bbatsov** [comment]
The commit message should start with `[Fix #1406`.

**Comment by Fabricationatorishnicatorist** [comment]
Done.

**Comment by bbatsov** [comment]
Thanks!

**Pull request merged**